src/jestTestAdapters/JestTestAdapterFactory.ts

```diff
@@ -0,0 +1,37 @@
+import JestTestAdapter from './JestTestAdapter';
+import JestPromiseAdapter from './JestPromiseTestAdapter';
+import JestCallbackAdapter from './JestCallbackTestAdapter';
+import * as semver from 'semver';
+
+export default class JestTestAdapterFactory {
+  public static getJestTestAdapter(loader?: NodeRequire): JestTestAdapter {
+    let jestTestAdapter: JestTestAdapter;
+
+    switch (this.getJestType(loader)) {
+      case 'promise':
+        jestTestAdapter = new JestPromiseAdapter;
```

Comment:
Why no `( )`? 😭

Reply:
I guess it's a matter of taste, but I am open for both notations. What do you want me to change it to?